frontend/src/scenes/pipeline/hogfunctions/metrics/HogFunctionEventEstimates.tsx

```diff
@@ -50,15 +50,17 @@ export function HogFunctionEventEstimates(): JSX.Element | null {
                 <>
                     {sparkline.count > EVENT_THRESHOLD_ALERT_LEVEL ? (
                         <LemonBanner type="warning">
-                            <b>Warning:</b> This destination would have triggered{' '}
+                            <b>Warning:</b> This {type === 'transformation' ? 'transformation' : 'destination'} would
```

Comment:
I don't think this warning is necessary at all for transforms... It only really applies to destinations where there is a load on the downstream service. Most transforms won't have filters and this indicates its an issue when it really isn't